### ISSUE TITLE
how to  implement Averaging Weights Leads to Wider Optima and Better Generalization in keras 

### ISSUE THREAD

**Issue by fryng** [fastai/fastai#583]
I'm currently implementing Averaging Weights Leads to Wider Optima and Better Generalization in keras ,the source code(Pytorch) is here [https://github.com/fastai/fastai/blob/master/fastai/swa.py]( ) , but i'm stucking in the **fix_batchnorm()** function. how could i update the params(moving_mean,moving_variance) in batchnorm layers by passing the inputs? In Pytorch, u just do like this `model(inputs)`, In keras , `model.predict(inputs)` doesn't work
